REST Bindings/openapi.yaml

```diff
@@ -231,11 +231,81 @@ paths:
           $ref: '#/components/responses/403ClientUnauthorized'
         '413':
           $ref: '#/components/responses/413CapturePayloadTooLarge'
+        '406':
+          $ref: '#/components/responses/406NotAcceptable'
         '415':
           $ref: '#/components/responses/415UnsupportedMediaType'
         '500':
           $ref: '#/components/responses/500ImplementationException'
-
+    get:
+      tags:
+        - 'Capture'
+      parameters:
+        - $ref: '#/components/parameters/NextPageToken'
+        - $ref: '#/components/parameters/PerPage'
+      summary: Returns a list of capture jobs.
+      description: |
+        An endpoint to list capture jobs.
+        This endpoint supports pagination.
+      responses:
+        '200':
+          headers:
+            GS1-EPCIS-Version:
+              $ref: '#/components/headers/GS1-EPCIS-Version'
+            GS1-EPCIS-Min:
+              $ref: '#/components/headers/GS1-EPCIS-Min'
+            GS1-EPCIS-Max:
+              $ref: '#/components/headers/GS1-EPCIS-Max'
+            GS1-CBV-Version:
+              $ref: '#/components/headers/GS1-CBV-Version'
+            GS1-EPCIS-Extensions:
+              $ref: '#/components/headers/GS1-EPCIS-Extensions'
+            GS1-CBV-Extensions:
+              $ref: '#/components/headers/GS1-CBV-Extensions'
+            Link:
+              description: |
+                A pagination header link. This header works together with the `perPage` and `nextPageToken` query string
+                parameters.
+                As long as there are more resources to retrieve, the `Link` header contains the URL of the next page and
+                the attribute `rel="next"`. The last page is indicated by the absence of the `rel="next"`.
+              schema:
+                $ref: '#/components/schemas/Link'
+            GS1-Next-Page-Token-Expires:
+              $ref: '#/components/headers/GS1-Next-Page-Token-Expires'
+          description: |
+            When EPCIS events are added through the capture interface, the capture process can run asynchronously.
+            A capture job resource exposes the state of the capture job to the client. This endpoint returns all
+            capture jobs that were created.
+          content:
+            application/json:
+              example: [
+                {
+                  "captureId": "id261378658356",
```

Comment:
Should it not be ID to be consistent with the rest of the spec?

Reply:
Seems like we are a bit inconsistent in the spec with IDs: an event has just an `id` a subscription has a `subscriptionId` we should choose one way to name IDs shouldn't we?